poh/src/poh_service.rs

```diff
@@ -237,6 +237,37 @@ impl PohService {
         }
     }
 
+    /// Loops to record current record and any newly received records
+    fn record(
```

Comment:
probably want to do a measurement to see if we should add inline annotations here - or if it doesn't make any difference.